Neos.Neos/Documentation/Appendixes/ReleaseNotes/430.rst

```diff
@@ -0,0 +1,141 @@
+=====
+4.3.0
+=====
+
+Automatic image variants generation
+===================================
+Automatically generate variants of an original image based on the newly introduced image variant presets configuration.
+
+In this first version, variants are automatically created as soon as an asset is created – for example, when it has been 
+uploaded by an editor or otherwise been imported. Future versions may defer this generation into a job queue, or provide a way 
+to put conditions of the generation of these variants.
+
+See the included documentation for configuration syntax and further background.
+
+This change also includes a basic view displaying all variants of an asset in the edit view of the Media Browser. 
+While this new view (implemented as a tab) may be useful already, it is neither especially polished nor does it provide additional features. 
+Therefore it is disabled by a feature switch for now. See the ``Settings.yaml`` of ``Neos.Media.Browser`` for details on enabling the tab.
+
+Add fallback from ``itemRenderer`` to ``content`` for ``Neos.Fusion:Map`` and derived prototypes
```

Comment:
I suggest to use a wording like this
```
Enable Loops in Fusion AFX without @children annotation
=======================================================

Fusion AFX uses the key ``content`` by default to render children. Since Collections or Loops 
use `ìtemRenderer`` to define the inner content ``@children=itemRenderer`` had to be declared 
in the past. To overcome this Loop and deriver prototypes now got a fallback from ``itemRenderer``
to content. This avoids a common error in afx code.
```